views/home.handlebars

```diff
@@ -1,79 +1,40 @@
-<div class="login-card">
```

Comment:
Hey. So if I push your changes through, everything I added will be deleted... is that what we want? @toTOEro

Reply:
It's moved to login.handlebars; that will let it render properly